### ISSUE TITLE
jobs in queue graph show number job=1 although there is no running job

### ISSUE THREAD

**Issue by HNguyenTruong** [UnivaCorporation/unisight-visualizations#23]
See my screenshot
<img width="1792" alt="Screen Shot 2022-02-04 at 9 32 48 AM" src="https://user-images.githubusercontent.com/30269233/154293343-7a4c0447-e741-4fa2-b8f2-b9e59e2a0603.png">

You can see all.q showing there is 1 job but actually there is no job running in the cluster. I am able to track the issue come from the logic of the function def jobs_per_queue(myquery): in file /opt/unisight-data-bridge/lib/python3.6/site-packages/unisightDataBridge/nodeexporter/metrics.py

the function get queue info from running jobs graphql query, and when jobs completed, the list return null and the function keep the last value of running job instead of update to 0
@cameronbrunner

**Comment by cameronbrunner** [comment]
@HNguyenTruong you reported this a while back to @kornelijaveseli directly, correct?  I am merging her changes that should address this issue.

**Comment by HNguyenTruong** [comment]
Right, @kornelijaveseli told me to file an issue in github to keep track the issue. Thanks for taking care of it.

**Comment by cameronbrunner** [comment]
I just tested out the dashboard with my local installation.  Looks good!

![image](https://user-images.githubusercontent.com/12917647/154309182-ebf08f45-2172-4dd7-84b8-925187d59781.png)

**Comment by cameronbrunner** [comment]
This should be addressed in #22
